app/controllers/groups_controller.rb

```diff
@@ -380,6 +380,34 @@ def add_members
     end
   end
 
+  def add_owners
+    group = Group.find_by(id: params.require(:id))
+    raise Discourse::NotFound unless group
```

Comment:
I know this is not your code here, but should we just do: `Group.find(params.required(:id))` if we want to fail right away? It seems like it would achieve the same behavior in one line instead of two.

```suggestion
    group = Group.find(params.require(:id))
```

Reply:
I'm not sure we handle `ActiveRecord::RecordNotFound`, while we have [this](https://github.com/discourse/discourse/blob/main/app/controllers/application_controller.rb#L223-L231) handler for `Discourse::NotFound`. 🤔

Reply:
I would say it's default rails behavior, but might be wrong on this

Reply:
I see we use `#find` in quite a few other places, including places where the author seemingly thinks it will return `false` if it fails. 😬 Will dig a bit deeper.

Reply:
I just tried it on topics_controller#show locally, doing `Topic.find(non_existing_id)` and it returns a 404. Anyways no strong feelings on this, I would rather rely on find failing but both are okay to me.

Reply:
I don't mind either one, but I think we should try to settle on one approach if possible. Changing this one in this PR won't change the state we're in with a mix of both, so I'll create a topic for this where we can discuss. 🙂